src/analytics/README.rst

```diff
@@ -15,19 +15,20 @@ To install frontend-analytics into your project::
 
 Add the following configuration code to your app::
 
-    import { configureAnalytics, initializeSegment } from '@edx/analytics';
+    import { configure as configureAnalytics, SegmentAnalyticsService } from '@edx/analytics';
```

Comment:
The `@edx/analytics`, `@edx/frontend-logging`, etc, packages should all be updated here, presumably?

Reply:
👍